src/Altinn.Platform/Altinn.Platform.Events/Tests/Data/apps/ttd/steffens-2020-v2/config/authorization/policy.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't see this app being included in any tests. No need for it's policy file.

Reply:
removed